src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/RuntimeBinder.cs

```diff
@@ -737,11 +737,7 @@ private void AddConversionsForArguments(ArgumentObject[] arguments)
                 // Make sure we're not setting ref for the receiver of a call - the argument
                 // will be marked as ref if we're calling off a struct, but we don't want 
                 // to persist that in our system.
-                bool isFirstParamOfCallOrInvoke = false;
-                if (i == 0 && IsBinderThatCanHaveRefReceiver(payload))
-                {
-                    isFirstParamOfCallOrInvoke = true;
-                }
+                bool isFirstParamOfCallOrInvoke = i == 0 && IsBinderThatCanHaveRefReceiver(payload);
```

Comment:
> intentionally left out

Yes, in cases like this there's no need for the parens, with precedence being obvious.